iwut.py

```diff
@@ -0,0 +1,223 @@
+from IPython.core.display import HTML, display
+from IPython import get_ipython
+from traceback import FrameSummary, linecache
+from html import escape
+import sys
+
+
+CSS = """
```

Comment:
For a first try at CSS in BP, this is pretty good! I'd say, however, that we should always try accounting for dynamically sized windows. That is, someone with an old 11 inch Del should have the same Jupyter NB experience as someone with the newest 16 inch Macbook Pro. To accomplish this, it's better to use percentages (or em, or rem) for any programmed spacing rather than pixel measurements. For example, `padding: 0 5px` might look something like `padding: 0 1%`. Take a look at [this article](https://medium.com/@madhum86/css-font-sizing-pixels-vs-em-vs-rem-vs-percent-vs-viewport-units-b1485716afe7) if you're curious about the different units for CSS spacing! Pick a unit that makes the most sense, then try subbing out the pixel measurements with the dynamically flexible unit.

Reply:
I just realized that you probably didn't write this and this is Alvin's code LOL. All good, seems like there are no complaints for iwut anyways. The above comment is still relevant if you do frontend work in the future though (: